### ISSUE TITLE
Support PHP 7.4

### ISSUE THREAD

**Issue by justin-schroeder** [stripe/stripe-php#896]
We’re running into issues getting the stripe SDK to work in PHP 7.4. The error produced is:

```
Trying to access array offset on value of type int /var/task/vendor/stripe/stripe-php/lib/StripeObject.php:433
```

This makes sense, because line 433 is using an array accessor in `StripeObject::toArray()` to determine if a property is public or private before serializing to an array:

```php
if ('_' === $k[0]) ...
```

However, in PHP 7.4 you should not use array notation to access the first character of a number like  in PHP 7.3.

```php
// In PHP 7.3
$x = 123;
echo $x[0]; // outputs NULL

// In PHP 7.4
$x = 123;
echo $x[0]; // throws Notice Trying to access array offset on value of type int in 
```

For environments that convert error/notices into exceptions (like Laravel) this causes an exception to be thrown.

**Comment by ob-stripe** [comment]
Thanks for the bug report @justin-schroeder, I was able to reproduce it easily.

Out of curiosity, how did you discover it? Are you using integers as metadata keys?

**Comment by justin-schroeder** [comment]
@ob-stripe yep. I suppose we're an odd one out in that sense ha. Sometimes we use numeric keys to "fake" a sortable array of properties.

**Comment by ob-stripe** [comment]
Just to be sure there's not another issue at play here, this can only happen if you call `toArray()` after setting metadata with integer keys but before calling `save()`, right? The API should always return strings for keys, even if integers were passed on the way in.

**Comment by justin-schroeder** [comment]
@ob-stripe actually no, the code was a pretty simple controller method (context is Laravel):

```
    public function getPlans(Request $request)
    {
        $all_plans = \Stripe\Plan::all();
        return response($all_plans);
    }
```

Some of the plans happen to have metadata with numeric keys.

**Comment by ob-stripe** [comment]
Just released stripe-php 7.27.3 with your fix, thanks again.

Would you mind sharing your account ID and one of the plan IDs so we can take a closer look at what happens on our end?

**Comment by justin-schroeder** [comment]
Awesome, thanks for the quick fix.

acct_1G1bAyAwqhErLVlI

plan_GqkWIob7obsABb
plan_GqkW2BWGqt1BEZ
